### PR TITLE
[backport/1.22] release: 1.22.1 patch stack

### LINKED ISSUE #21650: release note for v1.22.1/version_history/current is 404
image

### LINKED ISSUE #21652: Envoy Distroless Image for v1.22.1 isn't released
Distroless image for version v1.22.1 isn't released. 

The last tag for v1.22-latest is two months old. 
https://hub.docker.com/r/envoyproxy/envoy-distroless/tags

### PR DESCRIPTION
**Pull request by phlax**
Fix #21652 Fix #21650

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
the release/v1.22 branch reliably does/not fail with the disk space issue after/before 1.22.1 patch stack is applied

the build files - esp dwp - are considerably larger and that is multiplied in the docker build

not sure what we can do about the disk space - i guess we can try and find more things to remove before the run, or see what is still taking up space - i removed all the obviously unused things previously, but i was pretty careful, i guess i can do it more agressively

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.22.1-dev
+1.22.1
```
Comment (phlax) [inline VERSION.txt:1]: i think we should skip straight to 1.22.2 here

Comment (mattklein123) [reply VERSION.txt:1]: Yes +1, can you add a commit to do this? We will need a new tag.

Comment (phlax) [reply VERSION.txt:1]: yep, on it...

**Review by mattklein123 — APPROVED** [review]
Thank you so much!

**Comment by phlax** [comment]
force pushed minor formatting change to preserve commits

**Comment by Rs4178** [comment]
> Thank you so much!

Rs

**Pull request merged**